common/include/pcl/common/time.h

```diff
@@ -59,9 +59,7 @@ namespace pcl
   {
     public:
       /** \brief Constructor. */
-      StopWatch () : start_time_ (std::chrono::steady_clock::now())
-      {
-      }
+      StopWatch () = default;
```

Comment:
You mentioned this in the issue you opened on llvm-project. Do you still consider this a problem? The way with the default member initializer seems correct, no?

Reply:
Well, you can't simply write:
```
StopWatch () : start_time_ (std::chrono::steady_clock::now()) = default;
```
and simply replacing it with
```
StopWatch () = default;
```
would leave `start_time_` uninitialized (documentation is unclear about the default value of a `std::chrono::time_point<std::chrono::steady_clock>`)

The way I rewrote it, with direct member initialization, is correct.

Reply:
On further reflection, I'm probably over-thinking this;
```
StopWatch() = default;
```
likely does entirely the right thing. (The compiler is smarter than I am, no surprise.)

Reply:
But the direct member initialization of `start_time_` is still necessary, no? [The documentation](https://en.cppreference.com/w/cpp/chrono/time_point/time_point) says `time_point`'s "default constructor creates a time_point representing the Clock's epoch". It's not clear what `steady_clock`'s epoch is, [maybe time since the last reboot](https://en.cppreference.com/w/cpp/chrono/steady_clock), but it is very likely not equivalent to `std::chrono::steady_clock::now()`.
So the way you wrote it looks absolutely correct.